modules/nf-core/multiqc/main.nf

```diff
@@ -3,8 +3,8 @@ process MULTIQC {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/multiqc:1.19--pyhdfd78af_0' :
-        'biocontainers/multiqc:1.19--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/multiqc:1.22.2--pyhdfd78af_0' :
+        'biocontainers/multiqc:1.22.2--pyhdfd78af_0' }"
```

Comment:
You mention Seqera containers in the PR description for MultiQC. If you wanted to use Seqera wave containers here, you could use the following. Up to you which one to go with for the moment. I have tested the containers below locally and they worked.

```suggestion
        'oras://community.wave.seqera.io/library/pip_multiqc:dab7a0322e1851bb' :
        'community.wave.seqera.io/library/pip_multiqc:2c2e276ad8997cc4' }"
```

Reply:
I updated it in the module config as changing the main.nf from MultiQC would cause a failure with linting. I would prefer to do it here if possible, but it would anyways only be to circumvent the multiqc gitpod issue.